test/framework/manifest/gatewayroute.go

```diff
@@ -0,0 +1,71 @@
+package manifest
+
+import (
+	appmesh "github.com/aws/aws-app-mesh-controller-for-k8s/apis/appmesh/v1beta2"
+	"github.com/aws/aws-sdk-go/aws"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type GRBuilder struct {
+	Namespace string
+}
+
+func (b *GRBuilder) BuildGatewayRoute(instanceName string, vsName string, prefix string) *appmesh.GatewayRoute {
```

Comment:
Looks like this function builds a GR with HTTP route by default. Can we name it as such and create an equivalent function for GRPC? Or pass in the protocol as a parameter to this function and then split off..

Reply:
+1, will adjust